### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by SaadMahi**
Css Battle Skull Practice

### CODE REVIEW

**Review by OSAMA478 — APPROVED** [review]
perfectly handled with CSS

File: index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+      .teeth-2 {
+        transform: translate(15px, 30px);
+      }
+
+      .teeth-3 {
+        transform: translate(35px, 30px);
+      }
+
+      [class^="bone"] {
```
Comment (murtazshaikh) [inline index.html]: what this ^ represents in this selector

**Review by murtazshaikh — APPROVED** [review]
Amazing @SaadMahi

File: index.html (hunk 1 above)
Comment (SaadMahi) [reply index.html]: this is to target attributes value that has a starting letter of x.

**Pull request merged**